.github/workflows/nestbuildmatrix.yml

```diff
@@ -848,6 +847,5 @@ jobs:
         with:
           name: "${{ matrix.NEST_BUILD_TYPE }}-build-logs-${{ matrix.os }}-${{ matrix.cpp_compiler }}"
           path: |
-            install_manifest.txt
-            **.log
-            build/reports/**
+            build/install_manifest.txt
```

Comment:
Perfect. This then fixes the issue with the warning that those files were not found in the  provided path:

Reply:
@lekshmideepu Github now sees your positive comment as an unresolved, merge-blocking comment. Could you resolve it and approve, unless there are other issues?